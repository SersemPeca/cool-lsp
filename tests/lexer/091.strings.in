"escaped new line in string here: \
same string"
