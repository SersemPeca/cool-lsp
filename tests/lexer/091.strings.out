#2 STR_CONST "escaped new line in string here: \nsame string"
